src/activities/discussions/DiscussionForumEntity.js

```diff
@@ -1,11 +1,21 @@
-import { Actions } from '../../hypermedia-constants.js';
+import { Actions, Rels } from '../../hypermedia-constants.js';
 import { Entity } from '../../es6/Entity.js';
 import { performSirenActions } from '../../es6/SirenAction.js';
 
 /**
  * DiscussionForumEntity class representation of a D2L Discussion Forum.
  */
 export class DiscussionForumEntity extends Entity {
+	/**
+	 * @returns {string} the href of the collection that the discussion forum belongs to
+	 */
+	forumsHref() {
+		if (!this._entity || !this._entity.hasLinkByRel(Rels.Discussions.forums)) {
```

Comment:
Maybe I have to change the rel for this to work over here: https://github.com/Brightspace/lms/pull/28852#discussion_r996115944

Reply:
Yeah was gonna ask you what the rel should be here

Reply:
It is serialized like this: ![image](https://user-images.githubusercontent.com/15062048/195944514-64f22627-397a-4359-9539-8d44f40acf13.png)

Reply:
I've added the rel to match your siren entity!

Reply:
Thank you so much! 😄